### DISCUSSION TITLE
3 days passed,didn't get benefits.

### DISCUSSION THREAD

**Discussion by MuntasirSZN** [community/community#129607] [GitHub Education] [answered]
### Hi everyone,

I registered for student developer pack on june 21,and now its june 24.# days passed,haven't got my benefits.When I will get it.I got approved,by the way.
![image](https://github.com/community/community/assets/161931072/56d1b80d-8ae1-4e75-80b0-f82523dab655)

**Comment by davevad93** [comment, marked as answer]
Hi @MuntasirSZN ,

Don't worry, it may take a while, from a few days to a few weeks depending on different factors such as the volume of requests. When the benefits are available the status of your application will turn purple and change to:

`Congrats! Your request for @MuntasirSZN  was approved and your benefits are now available`

**Reply by MuntasirSZN** [reply]
Ok.Wait...Wait.....Wait.......
